src/api_server/src/lib.rs

```diff
@@ -243,7 +243,7 @@ impl ApiServer {
                 response
             }
             Err(err) => {
-                error!("{}", err);
+                error!("{:?}", err);
```

Comment:
Is it guaranteed that the Debug formatter will produce a sanitised output?

Reply:
We might want to have a test for the error message.